CONTRIBUTING.md

```diff
@@ -10,12 +10,18 @@ For information about contributing to the IntelliJ Platform itself, please visit
 Here are some useful things to know before authoring SDK content and submitting your Pull Request.
 * Dummy list item
 {:toc}
- 
+
 ## Setting Up the Documentation Build Environment
 
-This site runs via [Jekyll](https://jekyllrb.com), which is a popular static site generator, written in Ruby. It can be hosted locally to ensure that any changes are correct. Once set up, running the site is as easy as calling `rake preview`.
+This site runs via [Jekyll](https://jekyllrb.com), which is a popular static site generator, written in Ruby.
+It can be hosted locally to ensure that any changes are correct.
+Once set up, running the site is as easy as calling `rake preview`.
 
-Alternatively, the site can also be hosted in a [Docker container](https://www.docker.com). On Mac and Windows, this means the site is hosted in a virtual machine. Docker maintains this container, building it based on the instructions in the [`Dockerfile`](Dockerfile). All dependencies (Ruby, etc.) are automatically installed when building the image, which reduces the manual configuration steps. The Docker image is also used to build the [published site](https://www.jetbrains.org/intellij/sdk/docs/index.html), so it is a known working environment.
+Alternatively, the site can also be hosted in a [Docker container](https://www.docker.com).
+On Mac and Windows, this means the site is hosted in a virtual machine.
+Docker maintains this container, building it based on the instructions in the [`Dockerfile`](Dockerfile).
+All dependencies (Ruby, etc.) are automatically installed when building the image, which reduces the manual configuration steps.
+The Docker image is also used to build the [published site](https://www.jetbrains.org/intellij/sdk/docs/index.html), so it is a known working environment.
```

Comment:
Correct changes (here and elsewhere in this doc) per our style guide, but these aren't content changes. Is it wiser to postpone style cleanup until the technical content is correct? (IJSDK-252, IJSDK-575)